src/rand.rs

```diff
@@ -63,18 +63,17 @@ impl Rng {
         Self::default()
     }
 
-    #[cfg_attr(feature = "cargo-clippy", allow(should_implement_trait))]
-    pub fn next(&mut self) -> u32 {
+    pub fn gen_int(&mut self) -> u32 {
         self.pcg32()
     }
 
-    pub fn next_in_range(&mut self, min: u32, max: u32) -> u32 {
+    pub fn gen_int_interval(&mut self, min: u32, max: u32) -> u32 {
```

Comment:
That's an open-close interval, e.g. `r.gen_int_interval(7, 11)` can return 7, 8, 9, and 10, but *not* 11.  Is that what you want?

Reply:
Are you asking about the logic or the name?
I did not write that function, but I think this behaviour is fine.

Reply:
Yeah, open-close is fine, I was just making that's what we expected.  The name is fine.

Reply:
Might be worth adding a bit of doc to make this explicit.